rdrf/rdrf/templates/rdrf_cdes/generic_patient.html

```diff
@@ -170,7 +170,7 @@
 {% block content %}
     <div class="row">
         <div class="col-md-12">
-            <p><h3><span class="glyphicon glyphicon-list-alt" aria-hidden="true"></span> {% trans "Demographics" %}</h3></p>
+            <p><h3><span class="glyphicon glyphicon-list-alt" aria-hidden="true"></span> {% trans custom_form_name %}</h3></p>
```

Comment:
We don't know if it is a `custom_form_name` for sure. Just call it `form_title`?

Reply:
Updated.